packages/select-rich/src/LionSelectRich.js

```diff
@@ -660,7 +651,7 @@ export class LionSelectRich extends OverlayMixin(
   }
 
   _isEmpty() {
```

Comment:
`_isEmpty` exists in `ChoiceGroupMixin` with almost the same implementation. Should we remove it from here.

Reply:
Yep, if possible :)!